### PR TITLE
Proposal for issue #203

### PR DESCRIPTION
**Pull request by renatomefidf**

### CODE REVIEW

File: web/assets/css/main.css [hunk 1]
```diff
@@ -55,47 +55,49 @@ header img.logo {
...
 
 .ui.transparent.menu {
-  opacity: 0.90;
+    opacity: 0.90;
 }
 
 #phansible .container {
-  width: 1050px;
-  margin: 0px auto;
+    /*width: 1050px;*/
```
Comment (naxhh) [inline web/assets/css/main.css]: Please don't keep commented code.

**Comment by naxhh** [comment]
Looks ok! I'll test it on staging

Thanks!

**Comment by renatomefidf** [comment]
What's the staging url?

**Comment by naxhh** [comment]
http://staging.phansible.com/

But you shouldn't do your normal testing there.

Since I can re-deploy 5-6 different versions of the web in a short period of time.
So you'll never know if the version you are checking is the one you want or not.

**Comment by renatomefidf** [comment]
Thank you! No problem, I just wanted to do a overall looking!

**Comment by naxhh** [comment]
Of course! test as much as you want and report back all you can find :)

**Pull request merged**